### PR TITLE
feat: enable typescript biome rules

### PR DESCRIPTION
**Pull request by anonrig**
We weren't leveraging enough Typescript rules. Adding these caught some...

### CODE REVIEW

File: tests/sentry/api/endpoints/test_accept_organization_invite.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 from sentry.silo.safety import unguarded_write
 from sentry.testutils.cases import TestCase
 from sentry.testutils.factories import Factories
+from sentry.testutils.helpers.options import override_options
```
Comment (evanpurkhiser) [inline tests/sentry/api/endpoints/test_accept_organization_invite.py:22]: Looks like some stuff snuck in

Comment (anonrig) [reply tests/sentry/api/endpoints/test_accept_organization_invite.py]: Yup, removed.

File: static/app/views/performance/cache/referrers.ts [hunk 2]
```diff
@@ -1,6 +1,4 @@
-const PREFIX = `api.performance.cache`;
-
 export enum Referrer {
-  LANDING_CACHE_HIT_MISS_CHART = `${PREFIX}.landing-cache-hit-miss-chart`,
-  LANDING_CACHE_THROUGHPUT_CHART = `${PREFIX}.landing-cache-throughput-chart`,
+  LANDING_CACHE_HIT_MISS_CHART = `api.performance.cache.landing-cache-hit-miss-chart`,
+  LANDING_CACHE_THROUGHPUT_CHART = `api.performance.cache.landing-cache-throughput-chart`,
 }
```
Comment (scttcper) [inline static/app/views/performance/cache/referrers.ts:4]: stuff like this could probably be a const enum or just an `as const` object

**Review by scttcper — APPROVED** [review]

File: static/app/views/performance/cache/referrers.ts (hunk 2 above)
Comment (anonrig) [reply static/app/views/performance/cache/referrers.ts:4]: I raise your recommendation: I think we should just avoid having any `enum` and use JS objects to reduce typescript execution duration.

**Pull request merged**